ethstats/ethstats.go

```diff
@@ -515,10 +581,8 @@ func (s *Service) reportBlock(conn *websocket.Conn, block *types.Block) error {
 		"id":    s.node,
 		"block": details,
 	}
-	report := map[string][]interface{}{
-		"emit": {"block", stats},
-	}
-	return websocket.JSON.Send(conn, report)
+
+	return s.sendStats(conn, "block", stats)
```

Comment:
Nothing major but wondering if we should put these string literals (block/node-ping/etc) as consts up here https://github.com/celo-org/celo-blockchain/blob/aa3d9d5f5d175f78ef9f80bd32661fc25ad25eb0/ethstats/ethstats.go#L54